Assets/Scripts/Data/Pickup Pup/PPDataController.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/*
```

Comment:
Oh probably related to this error which is actually my fault. "WSerializationException: Type Dog is not marked as Serializable." The fix is trivial (just marking a var as NonSerialized). I'll do that and push to your branch here